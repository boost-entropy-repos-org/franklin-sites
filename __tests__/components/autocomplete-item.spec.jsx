import React from 'react';
import renderer from 'react-test-renderer';
import AutocompleteItem from '../../src/components/autocomplete-item';

describe('AutocompleteItem component', () => {
  test('should render', () => {
    const item = {
      pathLabel: 'A path to > some item',
      itemLabel: 'some item',
      value: 'item',
    };
    const active = false;
    const component = renderer.create(
      <AutocompleteItem
        item={item}
        active={active}
        substringToHighlight="item"
        key="key"
        handleOnClick={d => d}
      />,
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
