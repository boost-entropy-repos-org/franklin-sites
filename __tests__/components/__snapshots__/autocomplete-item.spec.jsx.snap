// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutocompleteItem component should render 1`] = `
<li>
  <button
    className=""
    onClick={[Function]}
    type="button"
  >
    A path to &gt; some 
    <b>
      item
    </b>
    
  </button>
</li>
`;
